funky-town-words/usr/share/asteroid-launcher/watchfaces/01.svg

```diff
@@ -0,0 +1,87 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
Just a comment for the future: It is a good habit to minimize your svg files (with something like http://www.svgminify.com/ ) if they are meant to be drawn often.

Reply:
Inkscape will always be a mystery for me :) I often find myself editing .svg files in vim instead, which is why I pay close attention to minimized xml.